unit_tests/test_plugins.py

```diff
@@ -13,6 +13,41 @@
 from charms.layer.jenkins import paths
 from charms.layer.jenkins.plugins import Plugins
 
+from systemfixtures.processes import Wget
+
+import io
+import argparse
+
+
+class Wget(Wget):
+
+    def __init__(self, locations=None):
+        self.locations = locations or {}
+
+    def __call__(self, proc_args):
+        cwd = proc_args.get("cwd") or ""
+        parser = argparse.ArgumentParser()
+        parser.add_argument("url")
+        parser.add_argument("-O", dest="output")
+        parser.add_argument("-q", dest="quiet", action="store_true")
+        parser.add_argument("-N", dest="timestamping", action="store_true")
+        parser.add_argument("--no-check-certificate", action="store_true")
+        args = parser.parse_args(proc_args["args"][1:])
+        content = self.locations[args.url]
+        file = args.url.split("/")[-1]
+        file_dir = os.path.join(cwd, file)
+        result = {}
+        if args.output == "-":
+            result["stdout"] = io.BytesIO(content)
+        elif (args.output is None):
+            with open(file_dir, "wb") as fd:
+                fd.write(content)
+        else:
+            with open(args.output, "wb") as fd:
+                fd.write(content)
+
+        return result
+
```

Comment:
This is in needed while the systemfixtures update is not ready.
Once the PR below gets merged and a new version of systemsfixtures gets published, this can be removed
https://github.com/testing-cabal/systemfixtures/pull/6

Reply:
Can you include this comment in the code itself? The PR comment will be lost once/if this is merged.

Reply:
My changes to systemfixtures have been merged and published so I'm removing this block.